templates/workspace_services/guacamole/user_resources/guacamole-azure-windowsvm/porter.yaml

```diff
@@ -126,10 +120,6 @@ upgrade:
         workspace_id: "{{ bundle.parameters.workspace_id }}"
         tre_id: "{{ bundle.parameters.tre_id }}"
         parent_service_id: "{{ bundle.parameters.parent_service_id }}"
-        arm_client_id: "{{ bundle.credentials.azure_client_id }}"
```

Comment:
Should arm_client_id and others still be in the uninstall though? Since it won't exist anymore...

Reply:
Good catch. Removed now.